src/vos/vos_internal.h

```diff
@@ -196,6 +196,8 @@ struct vos_pool {
 	int			vp_dying:1;
 	/** exclusive handle (see VOS_POF_EXCL) */
 	int			vp_excl:1;
+	/** this pool is for rdb */
+	bool			vp_rdb;
```

Comment:
See vp_excl, could be vp_rdb:1

Reply:
I'd keep it as separate 'bool', otherwise, the pool open count bits needs be reduced.